src/doc/src/getting-started/first-steps.md

```diff
@@ -7,7 +7,7 @@ $ cargo new hello_world --bin
 ```
 
 We’re passing `--bin` because we’re making a binary program: if we
-were making a library, we’d leave it off.
+were making a library, we’d pass `--lib`.
```

Comment:
I think `src/doc/src/guide/creating-a-new-project.md` also needs to be updated, since it talks about `--bin` in more detail.